ddl/ddl_db_change_test.go

```diff
@@ -433,7 +433,6 @@ func (s *testStateChangeSuite) TestShowIndex(c *C) {
 }
 
 func (s *testStateChangeSuite) TestParallelDDL(c *C) {
-	defer testleak.AfterTest(c)()
```

Comment:
Why remove this?

Reply:
testleak is verified in testStateChangeSuite.TeareDownSuite, call `testleak.AfterTest(c)` here make test unstable.